Added a recommendation for offline licensing

PeterSmithRedmond opened the pull request:
Note that the 'see also' url is en-us; I don't know the correct format for it.

PeterSmithRedmond committed:
Added a recommendation for offline licensing

Note that the 'see also' url is en-us; I don't know the correct format for it.

PRMerger14 commented:
@PeterSmithRedmond : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Edit pass

Changing voice to talk to the reader, rather than about them; sentence casing; removing `en-us/` from links; simplifying links for offline book; "Microsoft Store" rather than "store".

PRMerger7 commented:
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims reviewed (approved):
Looks good, thanks, Peter!
